home/src/main/resources/rdf/tbox/filegraph/ontologies.owl

```diff
@@ -12,7 +12,7 @@
     <owl:Ontology rdf:about="http://purl.org/ontology/bibo/"/>
     <owl:Ontology rdf:about="http://vivoweb.org/ontology/core"/>
     <owl:Ontology rdf:about="http://purl.obolibrary.org/obo/"/>
-    <owl:Ontology rdf:about="http://www.w3.org/2006/vcard/ns"/>   
```

Comment:
It appears that no substantive change is being made to this file. If that is true, please remove any changes to this file from the pull-request.